lib/mhv_ac/registration_form.rb

```diff
@@ -0,0 +1,40 @@
+# frozen_string_literal: true
+require 'common/models/form'
+
+module MHVAC
+  class RegistrationForm < Common::Form
+    attribute :icn, String
```

Comment:
@patrickvinograd you are correct.  Joe random user, e.g. you, me, etc can have LOA3 accounts.  In fact, I do and you should all too -- just go ahead and do it via Vets.gov.  Identity proofing is completely independent of any VA system.  It is only AFTER someone is identity proofed that we even make a connect to a VA system (e.g. MVI, in the future eMIS) to get any information about the user to then act upon (e.g. correlation IDs from MVI, veteran status from eMIS, etc).

@saneshark to your question - it's willy nilly if we add gates that the VA is not expecting; I am _incredibly_ deep in and familiar with the issues around identity and users w/in the VA writ large and I can absolutely confirm that it will be A Big Deal if we change gating methods, which is why that's not currently in the plans.  Also because of issues with identity and users and teh VA's own data, we do not plan to support delegates in the near future.  There are bigger issues the VA needs to work out before that becomes a plan or out side.

"We should not do account creation for users that are not LOA3. " That is absolutely correct.  In no way should MHV account creation be attempted for anyone that isn't already identity proofed.  @ayaleloehr you might want to jump in here on this.  

I would bring the question back to the original requirement -- which I don't know -- and the whole planned process by me as regards identity. I'm the identity SME, you know MHV's requirements, I can and should help you figure this out. @ayaleloehr would you agree?